src/app/project-sets/[projectSetId]/columns.tsx

```diff
@@ -0,0 +1,160 @@
+'use client'
+
+import {type ColumnDef} from "@tanstack/react-table"
+import {type ProjectRequirement, RequirementOperator} from "@/_temp_types/projects"
+import {DataTableColumnHeader} from "@/components/ui/data-table-column-header"
+import {
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuRadioGroup,
+    DropdownMenuRadioItem,
+    DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu"
+import React from "react"
+import {ChevronDownIcon} from "@radix-ui/react-icons"
+
+
+function convertAttributeName(projectId: number): string {
```

Comment:
nit;

pls let's just be consistent with using arrow functions, gives me the ick to have both `function` and arrow functions